sdk/graphrbac/graph/README.md

```diff
@@ -32,8 +32,10 @@ import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
 import { GraphRbacManagementClient, GraphRbacManagementModels, GraphRbacManagementMappers } from "@azure/graph";
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new GraphRbacManagementClient(creds, subscriptionId);
+msRestNodeAuth.interactiveLogin({{ tokenAudience: "https://graph.windows.net" }}).then((creds) => {
+  const client = new GraphRbacManagementClient(creds, subscriptionId, {
+    baseUri: "https://graph.windows.net"
+  });
```

Comment:
For now updating the sample https://github.com/Azure/azure-sdk-for-js/issues/3191